samples/drivers/display/boards/mimxrt595_evk_cm33.conf

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright 2023, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_HEAP_MEM_POOL_SIZE=1974272
+CONFIG_MCUX_LCDIF_EXTERNAL_FB_MEM=y
+# Use FlexSPI2 base address for framebuffer (pSRAM is present on this bus)
+CONFIG_MCUX_LCDIF_EXTERNAL_FB_ADDR=0x38000000
```

Comment:
I think the way to do this might be to make these displays Zephyr shields, what are your thoughts on this approach? I can either integrate the change into this PR, or create a follow up PR to move the radium RM68200 display to be a shield, whichever you prefer

Reply:
Hi @danieldegrasse ,
Creating shields makes sense to me.  A follow-up PR is fine with me, if that helps with getting this one approved.

Reply:
I'll create a follow up PR- this PR is already large, and I'd like to avoid adding additional changes to it that aren't required to get the hardware working